index.bs

```diff
@@ -522,6 +522,158 @@ See also:
 * [Security and privacy are essential](https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy)
 * [What data does this specification expose to an origin?](https://www.w3.org/TR/security-privacy-questionnaire/#underlying-platform-data)
 
+<h3 id="support-non-fully-active">Support non-"fully active" documents</h3>
+
+After a user navigated away from a document,
+the document might stay around in a non-"[=Document/fully active=]" state,
```

Comment:
I don't think there's a need to add "scare quotes" around every usage of the term. Maybe the first one, but in other cases just a normal hyperlink suffices.